code/pcgen-launch4j.cfg.xml

```diff
@@ -1,23 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <launch4jConfig>
   <dontWrapJar>true</dontWrapJar>
   <headerType>gui</headerType>
   <jar>pcgen.jar</jar>
-  <outfile>C:\Projects\pcgendev\code\pcgen.exe</outfile>
+  <outfile>Z:\Netbeans\pcgen\code\pcgen.exe</outfile>
```

Comment:
This file changes when you use it with launch4j jar. This file is not used by the build script and will be removed once testing of the script generated one are complete.

I do use Netbeans and I'm aware others do as well as I see related questions/comments on the email traffic.